.github/workflows/deploy-to-openshift-backend-efx.yml

```diff
@@ -118,15 +124,15 @@ jobs:
           oc create imagestream ${{ env.IMAGE_NAME }} 2> /dev/null || true && echo "backend image stream in place"
 
           # Create tag for UAT env from DEV env image
-          oc tag ${{ env.NAMESPACE }}-dev/${{ env.REPO_NAME }}-${{ env.APP_NAME_BACKEND }}:${{ steps.get-latest-tag.outputs.tag }} ${{ env.NAMESPACE }}-test/${{ env.REPO_NAME }}-${{ env.APP_NAME_BACKEND }}:${{ steps.get-latest-tag.outputs.tag }}
+          oc tag ${{ env.NAMESPACE }}-dev/${{ env.REPO_NAME }}-${{ env.APP_NAME_BACKEND }}:${{ inputs.tag }} ${{ env.NAMESPACE }}-test/${{ env.REPO_NAME }}-${{ env.APP_NAME_BACKEND }}:${{ inputs.tag  }}
```

Comment:
Whitespace, no biggie.

Reply:
(Resolving as you might fix this before merging but it sure doesn't need to hold the PR up)